stubs/inertia/resources/js/Pages/OAuth/Partials/OAuthAppManager.vue

```diff
@@ -0,0 +1,318 @@
+    name: '',
+    redirect_uris: [],
+    confidential: false,
+});
+
+const updateOAuthAppForm = useForm({
+    name: '',
+    redirect_uris: [],
+});
+
+const deleteOAuthAppForm = useForm({});
+
+const displayingClientCredentials = ref(false);
+const oauthAppBeingManaged = ref(null);
+const oauthAppBeingDeleted = ref(null);
+
+const createOAuthApp = () => {
+    createOAuthAppForm.post(route('oauth-apps.store'), {
+        preserveScroll: true,
+        onSuccess: () => {
+            displayingClientCredentials.value = true;
+            createOAuthAppForm.reset();
+        },
+    });
+};
+
+const manageOAuthApp = (app) => {
+    updateOAuthAppForm.name = app.name;
+    updateOAuthAppForm.redirect_uris = app.redirect_uris;
+    oauthAppBeingManaged.value = app;
+};
+
+const updateOAuthApp = () => {
+    updateOAuthAppForm.put(route('oauth-apps.update', oauthAppBeingManaged.value), {
+        preserveScroll: true,
+        preserveState: true,
+        onSuccess: () => (oauthAppBeingManaged.value = null),
+    });
+};
+
+const confirmOAuthAppDeletion = (app) => {
+    oauthAppBeingDeleted.value = app;
+};
+
+const deleteOAuthApp = () => {
+    deleteOAuthAppForm.delete(route('oauth-apps.destroy', oauthAppBeingDeleted.value), {
+        preserveScroll: true,
+        preserveState: true,
+        onSuccess: () => (oauthAppBeingDeleted.value = null),
+    });
+};
+</script>
+
+<template>
+    <div>
+        <!-- Register OAuth App -->
+        <FormSection @submitted="createOAuthApp">
+            <template #title>
+                Register OAuth App
+            </template>
+
+            <template #description>
+                You may register an OAuth client to use our application's API.
+            </template>
+
+            <template #form>
+                <!-- App Name -->
+                <div class="col-span-6 sm:col-span-4">
+                    <InputLabel for="name" value="Application Name" />
+                    <TextInput
+                        id="name"
+                        v-model="createOAuthAppForm.name"
+                        type="text"
+                        class="mt-1 block w-full"
+                        autofocus
+                        autocomplete="off"
+                    />
+                    <InputError :message="createOAuthAppForm.errors.name" class="mt-2" />
+                </div>
+
+                <!-- Authorization Redirect URI -->
+                <div class="col-span-6 sm:col-span-4">
+                    <InputLabel for="redirect_uri" value="Authorization Redirect URI" />
+                    <TextInput
+                        id="redirect_uri"
+                        v-model="createOAuthAppForm.redirect_uris[0]"
+                        type="url"
+                        class="mt-1 block w-full"
+                        autocomplete="off"
+                    />
+                    <InputError :message="createOAuthAppForm.errors.redirect_uris" class="mt-2" />
```

Comment:
Same goes for this line:
`<InputError :message="createOAuthAppForm.errors.createClient.redirect_uris" class="mt-2"/>`